scripts/build-profiles/stencil-component/commands/e2e.js

```diff
@@ -0,0 +1,51 @@
+const { execSync } = require('child_process');
+const { info } = require('@okta/scripts.logger');
+
+exports.command = 'e2e';
+exports.desc = 'E2E test Stencil components with Puppeteer and Jest';
+exports.builder = (yargs) => {
+  const epilogue = `
+The Stencil test runner uses Jest under the hood, so you can use any Jest CLI
+args and it will be passed through:
+
+  https://jestjs.io/docs/en/cli.html#options
+
+It also has some Stencil specific options, which can be found by looking at
+ARG_OPTS in this file:
+
+  https://github.com/ionic-team/stencil/blob/master/src/cli/parse-flags.ts
+  `;
+
+  return yargs
+    .example('yarn e2e --no-headless', 'Run with the browser in non-headless mode')
+    .example('yarn e2e -t o-help-link', 'Run only tests that match "o-help-link"')
+    .example('yarn e2e --updateSnapshot', 'Update snapshots after changing component structure')
+    .epilogue(epilogue);
+};
+
+exports.handler = () => {
+  const packageDir = process.cwd();
+  const pkgJson = require(`${packageDir}/package.json`);
+  const suiteName = `${pkgJson.name.replace('@okta/', '')}`;
+  const reportsDir = process.env.REPORTS_DIR || `${packageDir}/test-reports`;
```

Comment:
We likely will not use Bacon for this - so we will need to hook into Travis CI to ensure these are reported correctly.